test/test_MainWindowWithSessionManagement.py

```diff
@@ -253,7 +238,7 @@ def tearDown(self):
         shutil.rmtree("Test Folder")
 
 
-@unittest.skipIf(skip_as_conda_build, "On conda builds do not do any test with interfaces")
+@skip_ci
 class TestMainWindowWithSessionManagementCreateLoadSessionDialog(unittest.TestCase):
```

Comment:
btw there was a pre-existing error with this test:

```python
assert isinstance(dialog, eqt.ui.SessionDialogs.LoadSessionDialog)
```

complains because `eqt.ui.SessionDialogs.LoadSessionDialog` is monkey-patched to be a function rather than a class. Could fix it in a separate PR.

Reply:
Ok, we should open an issue!